test/integration/framework/framework.go

```diff
@@ -134,6 +135,24 @@ func DeleteNamespace(ctx context.Context, c client.Client, ns *corev1.Namespace)
 	return nil
 }
 
+func DeleteRuntimeClass(ctx context.Context, c client.Client, runtimeClass *nodev1.RuntimeClass) error {
+	if runtimeClass != nil {
+		if err := c.Delete(ctx, runtimeClass); err != nil && !apierrors.IsNotFound(err) {
+			return err
+		}
+	}
+	return nil
+}
+
+func DeleteWorkload(ctx context.Context, c client.Client, wl *kueue.Workload) error {
```

Comment:
This is already part of `DeleteNamespace`

Reply:
Deleted.